src/index.ts

```diff
@@ -137,6 +137,8 @@ export type {
   SelectMenuTabPanelProps,
   SelectMenuLoadingAnimationProps
 } from './SelectMenu'
+export {SelectPanel} from './SelectPanel'
+export type {SelectPanelProps} from './SelectPanel'
```

Comment:
not blocker, up to you: should we skip adding SelectPanel v1 to the main bundle and directly add v2. One less deprecation?